standards/technical-approach.md

```diff
@@ -0,0 +1,29 @@
+# Technical Approach Documents
+
+Technical Approach Documents (TADs) serve to help the engineering team document solutions to feature development, share ideas between teams and help highlight potential issues with tasks before they are encountered during the development process.
+
+TADs should strive to describe the whats, hows and whys of the technical problem that is trying to be solved. It should strive to understand the problem at hand in a comprehensive way and explore any challenges so that engineers can be confident in their implementations of solutions. In short writing a TAD should answer any technical questions prior to the start of actual development work, freeing up developers to focus on implementation once the major questions have been resolved.
+
+## Process
+
+TADs do not have to be created for all development work, but MUST be created in the following cases:
+
+- If a Business Requirements Document (BRD) has been created
```

Comment:
Can you say _who_ is expected to write a BRD in a given team/project? That way, a developer knows who is responsible for it and who they will get the BRD from.

Reply:
absolutely, thanks for pointing out that that part is missing